services/analytics/queries.py

```
@@ -88,12 +89,24 @@
-    WHERE cl.label_id IN ({reporter_label_ids})
-        AND cl.created_at >= DATE_TRUNC('week', CURRENT_DATE) - INTERVAL '1 week'
+GET_WEEKLY_REPORTER_CONVERSATION = text(f"""
+    WITH previous_texters AS (
+        SELECT DISTINCT from_field as phone_number
+        FROM twilio_messages
+        WHERE is_broadcast_reply = false
+            AND from_field != '{BROADCAST_SOURCE_PHONE_NUMBER}'
+            AND created_at >= DATE_TRUNC('week', CURRENT_DATE) - INTERVAL '1 week'
+    )
+    SELECT COUNT(DISTINCT tm.to_field)
+    FROM twilio_messages tm 
```

Ruff (0.8.2)

Trailing whitespace

Remove trailing whitespace

(W291)